spaces/gem/skin.scss

```
@@ -0,0 +1,9 @@
+.sushi {
```

The css selector should be identical to the space key.
```suggestion
.gem {
```

---

addressed